hexrd/ui/hexrd_config.py

```diff
@@ -736,10 +736,17 @@ def intensity_corrected_images_dict(self):
         instr = create_hedm_instrument()
 
         if HexrdConfig().apply_pixel_solid_angle_correction:
+            sangle = dict.fromkeys(images_dict.keys())
+            mi = 1E12 # some large number
+            # normalize by minimum of the entire instrument
+            # not each detector individually
             for name, img in images_dict.items():
                 panel = instr.detectors[name]
-                images_dict[name] = img / panel.pixel_solid_angles
-
+                sangle[name] = panel.pixel_solid_angles
+                mi = np.min((mi, sangle[name].min()))
+            for name, img in images_dict.items():
+                images_dict[name] = mi * img / sangle[name]
+            del sangle
```

Comment:
You don't need to delete `sangle` here. It will be deleted automatically when it goes out of scope.